scrapy/utils/display.py

```diff
@@ -3,20 +3,60 @@
 """
 
 import sys
+from distutils.version import LooseVersion as parse_version
+from platform import version
 from pprint import pformat as pformat_
 
 
 def _colorize(text, colorize=True):
     if not colorize or not sys.stdout.isatty():
         return text
     try:
-        from pygments import highlight
         from pygments.formatters import TerminalFormatter
+        from pygments import highlight
         from pygments.lexers import PythonLexer
-        return highlight(text, PythonLexer(), TerminalFormatter())
     except ImportError:
         return text
 
+    # All Windows versions >= "10.0.14393" interpret ANSI escape sequences
+    # using terminal processing.
+    #
+    # Enable enivornment variable `ENABLE_VIRTUAL_TERMINAL_PROCESSING`
+    # to activate terminal processing.
+    if sys.platform == "win32" and parse_version(version()) >= parse_version("10.0.14393"):
+        try:
+            import ctypes
+            kernel32 = ctypes.windll.kernel32
+            handle = kernel32.GetStdHandle(-11)
+            # set `ENABLE_VIRTUAL_TERMINAL_PROCESSING` flag
+            if not kernel32.SetConsoleMode(handle, 7):
+                raise ValueError
+        except ValueError:
+            return text
```

Comment:
@Gallaecio  please suggest how should I cover this part. I tried mocking windows platform but I think `parse_version` is still failing.

Reply:
Could you please push the test code that fails, so that I can have a look?

Reply:
I have pushed the test code.Please have a look, windows mocking part is failing giving this error `AttributeError: <module 'ctypes' from '/usr/lib/python3.6/ctypes/__init__.py'> does not have the attribute 'windll'
`